aten/src/ATen/native/native_functions.yaml

```diff
@@ -37,19 +37,31 @@
   use_c10_dispatcher: full
   variants: function
 
+# Computes the gradient of current tensor w.r.t. graph leaves.
 - func: backward(Tensor self, Tensor? gradient=None, bool keep_graph=False, bool create_graph=False) -> ()
```

Comment:
Does this method still exists? You remove it from Variable below?

Reply:
backward() always existed on Tensor; previously Variable overrode with its own definition (very naughtily, I might say: it's not a virtual function). When I delete on Variable, the old definition on Tensor is now the one exposed.